src/_pytest/runner.py

```diff
@@ -227,13 +227,18 @@ def pytest_runtest_makereport(item, call):
         outcome = "passed"
         longrepr = None
     else:
+        from _pytest.config import UsageError
```

Comment:
right now usageerror is something that should exit pytest, if this is applied as is, its going to be wrong

we might want to declare a new outcome for this

Reply:
I agree!

Any suggestions about the name of the new exception? I also think it should be public so it can be used by plugin authors.

About the outcome, can we use `"error"`?

Reply:
please start a discussion on the ml, this needs some more thinking

Reply:
Sent. 👍